### PR TITLE
Updating kibana to store session and oauth secrets for reuse, fix oau…

### PR DESCRIPTION
**Pull request by ewolinetz**
…thclient generation for ops

### CODE REVIEW

**Comment by ewolinetz** [comment]
aos-ci-test

File: roles/openshift_logging_kibana/tasks/main.yaml [hunk 1]
```diff
@@ -43,6 +43,31 @@
...
+  register: session_secret_file
+
+- name: Checking for oauth_secret
+  stat: path="{{generated_certs_dir}}/oauth_secret"
+  register: oauth_secret_file
+
+# gen session_secret if necessary
+- name: Generate session secret
+  copy:
+    content: "{{ 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789' | random_word(200) }}"
```
Comment (jcantrill) [inline roles/openshift_logging_kibana/tasks/main.yaml]: This can be replaced with oo_random_word

**Comment by ewolinetz** [comment]
aos-ci-test

**Comment by ewolinetz** [comment]
made recommended changes to use oo_random_word filter

**Comment by ewolinetz** [comment]
aos-ci-test

**Comment by ewolinetz** [comment]
groovy - [merge]

**Pull request merged**